Doc/library/functions.rst

```diff
@@ -1307,9 +1307,10 @@ are always available.  They are listed here in alphabetical order.
 
 .. function:: print(*objects, sep=' ', end='\\n', file=sys.stdout, flush=False)
 
-   Print *objects* to the text stream *file*, separated by *sep* and followed
-   by *end*.  *sep*, *end*, *file* and *flush*, if present, must be given as keyword
-   arguments.
+   Print *objects* to the text stream *file* (the standard output by default),
+   separated by *sep* (a space by default) and followed by *end*
+   (a newline by default).
+   *sep*, *end*, *file* and *flush*, if present, must be given as keyword arguments.
```

Comment:
As a Python expert, I dislike such redundant documentation: the default value is already documented on the " .. function:: print(*objects, sep=' ', end='\\n', file=sys.stdout, flush=False)" line. Moreover, this line ("*objects, " part) also explains that these parameters are keyword-only (cannot be passed as positional arguments).

Reply:
I understand where you're coming from, I personally feel it's clearer that way

Also I followed your advice when you told me to look into bugs.python.org for doc bugs and I did, it's on you 😉